src/teleop/gui/src/components/AutonWaypointEditor.vue

```diff
@@ -281,6 +329,49 @@ export default {
       setOdomFormat: "setOdomFormat",
     }),
 
+    sendEnableAuton() {
+      let course;
+
+      // If Auton Enabled send course
+      if (this.autonEnabled) {
+        course = {
+          enable: true,
+          // Map for every waypoint in the current route
+          waypoints: _.map(this.route, (waypoint) => {
+            const lat = waypoint.lat;
+            const lon = waypoint.lon;
+
+            // Return a GPSWaypoint.msg formatted object for each
+            return {
+              latitude_degrees: lat,
+              longitude_degrees: lon,
+              // WaypointType.msg format
+              type: {
+                val: waypoint.gate
+                  ? WAYPOINT_TYPES.GATE
+                  : waypoint.post
+                  ? WAYPOINT_TYPES.POST
+                  : WAYPOINT_TYPES.NO_SEARCH,
+              },
+              id: parseFloat(waypoint.id),
+            };
+          }),
+        };
+      } else {
+        // Else send false and no array
+        course = {
+          enable: false,
+          waypoints: [],
+        };
+      }
+
+      const course_request = new ROSLIB.ServiceRequest({
+        enableMsg: course,
+      });
+
+      this.course_pub.callService(course_request, () => {});
```

Comment:
Do you not need to at least say (res) => {}?

Reply:
Nah it works without it, iirc ESLint also complained if I included it since it didn't get used.